explainer.md

```diff
@@ -77,5 +184,32 @@ domain of the web app include this extension. Browser would refuse to
 load web apps without Source Code Transparency from a domain using a
 certificate with this extension.
 
+```
+# jka commentary
+This seems OK, although I'd note that many web clients don't use Transport
+Layer Security (TLS - also known as HTTPS), required for X.509 certificates.
+
+Certainly the most popular websites do use TLS -- I would argue that they have
+an incentive not to allow any of their competitors or new market entrants to
+inspect or learn from much, if any, of their web traffic.  Those clients and
+applications could be excluded from integrity guarantees as a result.
```

Comment:
I don't see TLS as being an anti-competitive measure, as competitors can still receive all the content of the website simply by visiting it, same as if TLS wasn't used. Spying on visitor's web traffic, on the other hand, is not something that competitors should normally be able to do, unless they run an ISP.